src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you can specify a generic font family like 'sans-serif' as a fallback.